src/jit/compiler.h

```diff
@@ -275,9 +275,7 @@ class  LclVarDsc
     unsigned char       lvOverlappingFields :1;  // True when we have a struct with possibly overlapping fields
     unsigned char       lvContainsHoles     :1;  // True when we have a promoted struct that contains holes
     unsigned char       lvCustomLayout      :1;  // True when this struct has "CustomLayout"
-#if FEATURE_MULTIREG_ARGS_OR_RET
-    unsigned char       lvIsMultiRegArgOrRet:1; // Is this argument variable holding a value passed or returned in multiple registers?
-#endif
+    unsigned char       lvIsMultiRegArgOrRet:1;  // Is this a variable something that would be passed or returned in multiple registers?
```

Comment:
Is this an unrelated change? Why would you remove the conditional compilation for this?